heartbeat/beater/manager.go

```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/libbeat/processors"
 
+	pkgerrors "github.com/pkg/errors"
```

Comment:
Imports fun :-)

@urso I assume you separate libbeat and heartbeat in the past in the imports. Is this something you think ew should do? I think in most place we have all the beats imports together.